core/src/main/java/com/linecorp/armeria/server/DefaultServiceRequestContext.java

```diff
@@ -159,6 +160,7 @@ private DefaultServiceRequestContext(
             long requestStartTimeMicros) {
 
         super(meterRegistry, sessionProtocol,
+              requireNonNull(routingContext, "routingContext").uuid(),
               requireNonNull(routingContext, "routingContext").method(), routingContext.path(),
```

Comment:
`requireNonNull(routingContext, "routingContext").method()` -> `requireNonNull(routingContext).method()` because we don't want to check it twice?